versioned_docs/version-8.6/self-managed/identity/troubleshooting/troubleshoot-identity.md

```diff
@@ -83,9 +83,9 @@ these options:
 1. Configure the communication between the services (for example in a cluster) to use IP ranges that fall within the
    ranges that Keycloak expects.
 2. If configuring the IP ranges is not an option, it is possible to disable the SSL requirement in Keycloak itself by completing the following steps:
-   1. In the `master` realm, set `Require SSL` to `none` by following the steps in [SSL modes](https://www.keycloak.org/docs/23.0.1/server_admin/#_ssl_modes).
```

Comment:
> I feel like those Keycloak docs are always updated too fast for the version to stay relevant - using /latest is probably the right way to go. Thank you! 🚀

My only hesitation is around version mapping what we support. In 8.6, it looks like we support Keycloak 24.x and 25.x. So this is still not quite right as-is, but I'm not sure we can just link to latest. 🤔